scripts/telegram_ros_bridge

```diff
@@ -65,19 +67,27 @@ class TelegramROSBridge(object):
     def _telegram_start_callback(self, _, update):
         """
         Called when a telegram user sends the '/start' event to the bot, using this event, the bridge can be connected
-        to a specific conversation
+        to a specific conversation.
+        This checks whether the user is on the whitelist if the whitelist is a list and allows any user if whitelist is False
+
         :param update: Received update event that holds the chat_id and message data
         """
-        if self._telegram_chat_id is not None and self._telegram_chat_id != update.message.chat_id:
-            rospy.logwarn("Changing to different chat_id!")
-            self._telegram_updater.bot.send_message(self._telegram_chat_id,
-                                                    "Lost ROS bridge connection to this chat_id (somebody took over)")
-
-        self._telegram_chat_id = update.message.chat_id
-
-        rospy.loginfo("Starting telegram ROS bridge for chat id {}".format(self._telegram_chat_id))
-        update.message.reply_text(
-            "Telegram ROS bridge initialized, only replying to chat_id {} (current). Type /stop to disconnect".format(self._telegram_chat_id))
+        
+        # Whitelist defaults to False to disable it and allow anyone. To enable the whitelist, set it to an (empty) list
+        whitelist = rospy.get_param(WHITELIST, False)
+        if whitelist == False or isinstance(whitelist, list) and update.message.from_user.id in whitelist:
+            if self._telegram_chat_id is not None and self._telegram_chat_id != update.message.chat_id:
+                rospy.logwarn("Changing to different chat_id!")
+                self._telegram_updater.bot.send_message(self._telegram_chat_id,
+                                                        "Lost ROS bridge connection to this chat_id (somebody took over)")
```

Comment:
`Lost ROS bridge connection to this chat_id`. Is this message send to the one who lost connection or to the one who is taking over?
In case it is send to the one who lost connection, it is useful to print the id of the person who took over. Which is not done right now. In case this is send to the one who takes over, chat id is not interesting.